src/renderer/store/index.js

```diff
@@ -1,11 +1,279 @@
+const bitcoin = require('bitcoin')
+
 import Vue from 'vue'
 import Vuex from 'vuex'
 
 import modules from './modules'
 
 Vue.use(Vuex)
+let vue = new Vue()
 
 export default new Vuex.Store({
   modules,
-  strict: process.env.NODE_ENV !== 'production'
+  strict: process.env.NODE_ENV !== 'production',
+  state: {
+    addresses: [],
+    transactions: [],
+    totalBalance: 0,
+    unconfirmedBalance: 0,
+    availableBalance :0,
+    tBalance: 0,
+    zBalance: 0,
+    blockHeight: 'Scanning',
+    peerCount: 'None',
+    walletPolling: false,
+    rpcCredentials : {
+      user : "",
+      password : "",
+      port : 0
+    },
+    contacts:[
```

Comment:
Each contact should be associated to a "chain" internally, since we will support HUSH + BTCH as our first two chains. We can just hardcode every contact having `chain="hush"` for now. This is so we can support 2 contacts named Bob, one on Hush and one on BTCH, for instance.